_data/gallery.yml

```diff
@@ -227,7 +227,13 @@
     name: Hangman
     image: hangmanc.png
     author: Sawyer Charles  
+  
   -
     name: File Metadata
     image: filemetadataanshuman73.png
-    author: Anshuman Agarwal 
\ No newline at end of file
+    author: Anshuman Agarwal 
+
+  -
+    name: PHP App: Hello Name
```

Comment:
I am guessing the build is failing owing to `:` in `name: PHP App: Hello Name`

The **build failure** message reads as: `"jekyll 2.4.0 | Error:  (/home/travis/build/fossasia/gci15.fossasia.org/_data/gallery.yml): mapping values are not allowed in this context at line 237 column 18"`